### PR TITLE
BAH 2962 | System user and templates 

### PR DESCRIPTION
**Pull request by atish160384**
Templates for appointment sms and registration sms and a system user with privilege of send sms added

### CODE REVIEW

File: masterdata/configuration/liquibase/liquibase.xml [hunk 1]
```diff
@@ -19,6 +19,51 @@
             INSERT INTO provider (person_id, identifier, creator, date_created, uuid, name) VALUES (@person_id, 'registration', 1, now(), uuid(), 'registration');
         </sql>
     </changeSet>
+    <changeSet id="BAH-2962-20230629" author="Anubhav" runAlways="true">
```
Comment (gsluthra) [inline masterdata/configuration/liquibase/liquibase.xml]: author should be "Bahmni" for all liquibase changesets.

File: masterdata/configuration/liquibase/liquibase.xml [hunk 2]
```diff
@@ -19,6 +19,51 @@
...
+        <comment>Create System SMS role and respective privilege for logging into bahmni by system</comment>
+        <sql>
```
Comment (gsluthra) [inline masterdata/configuration/liquibase/liquibase.xml]: There is no "check" for when to run. And it says, runAlways.. which means this insert will get added on each run?

File: masterdata/configuration/liquibase/liquibase.xml [hunk 3]
```diff
@@ -19,6 +19,51 @@
...
+            </sqlCheck>
+        </preConditions>
+        <comment>Adding bahmni-system-user</comment>
+        <sql>
+            SET @puuid = uuid();
+            INSERT INTO person(gender, birthdate_estimated, dead, creator, date_created, uuid) VALUES('M', 0, 0, 1, now(), @puuid);
+            SELECT person_id INTO @person_id from person where uuid = @puuid;
+            INSERT INTO person_name(person_id, preferred, given_name, family_name, creator, date_created, uuid) VALUES(@person_id, 1, 'System', 'SMS', 1, now(), @puuid);
+            INSERT INTO users(system_id, creator, date_created, person_id, uuid, username, password, salt) VALUES ('SystemUser', 1, now(),@person_id, uuid(), 'systemuser', 'e28e3ae84c66bfba6b2c50a406567f3e34fea1d76b17c006931571fe5d940f6c6b81e49cf8ea5e0adfca19fe3beb68d8ad79f7c3812e92b8d502a9afcf2029b2', '1c9d7e94aeeb7a2459ef45ed200b2944582e0e7088d75f9b57a3644861ea766c20a269b3fe2eadaff1bc445ecfbd9bd3c0c550dfd813de48d39423cd3d1a8b10');
+            INSERT INTO provider (person_id, identifier, creator, date_created, uuid, name) VALUES (@person_id, 'systemuser', 1, now(), uuid(), 'SystemUser');
```
Comment (gsluthra) [inline masterdata/configuration/liquibase/liquibase.xml]: can we instead call the user: `system-user`

**Review by gsluthra — APPROVED** [review]

**Review by rahu1ramesh — APPROVED** [review]

File: masterdata/configuration/roles/roles.csv [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+7e8bd096-da70-11ec-9d64-0242ac120002,FrontDesk,"User have ability to login and perform front desk operations(search patient, register patient etc)",Provider;Lab;Appointments:FullAccess,Edit Users;Get Providers;Get Users;Add Encounters;app:registration;Get Concepts;Get Encounters;Get Patients;Get People;Get Visits;Get Visit Types;View Patients;Add Patients;Edit Patients;Edit Patient Identifiers;Get Patient Identifiers;Add Visits;Edit Observations;Add Observations;Edit Encounters;Get Observations;Get Concept Sources;Edit Visits;Get Encounter Roles;Get Visit Attribute Types;app:document-upload;app:patient-documents;app:appointments;app:reports;app:common:closeVisit;Get Care Settings;Send Appointment Reminder SMS;Send Registration SMS;Send Appointment Booking SMS
```
Comment (angshu) [inline masterdata/configuration/roles/roles.csv:3]: where are you creating these privileges? I hope they are defined in config.xml of individual omods?

**Review by angshu — APPROVED** [review]

**Review by gsluthra — CHANGES_REQUESTED** [review]
Please incorporate comments discussed today. Users dont need privilege for SMS sending, since this is a system feature. When this feature is ON, the SMS MUST go, irrespective of who the user is.

**Comment by atish160384** [comment]
Sure Gurpreet will remove the creation of new roles and privillages as discussed other code will remain the same .Changes are now moved to a new PR.

**Pull request closed without merging**